app/controllers/concerns/saml_idp_auth_concern.rb

```diff
@@ -51,13 +51,18 @@ def store_saml_request
   end
 
   def requested_authn_context
```

Comment:
I meant can we make this method name plural now that it always returns an array 😁